nix/gce.nix

```diff
@@ -211,6 +218,12 @@ let
     };
   };
 
+  nixosVersion = builtins.substring 0 5 (config.system.nixos.version or config.system.nixosVersion);
```

Comment:
Just noting this is very similar to how nixops-aws works: https://github.com/NixOS/nixops-aws/blob/master/nix/ec2.nix#L166